Doc/tutorial/inputoutput.rst

```diff
@@ -283,7 +283,11 @@ two arguments: ``open(filename, mode)``.
 
 ::
 
-   >>> f = open('workfile', 'w')
+    try:
+        f = open('workfile', 'w')
+    except OSError as e:
+        # File open failed
+        ...
```

Comment:
Minor nitpicks: reST documentation should be 3 spaces for indentation (https://devguide.python.org/documenting/#use-of-whitespace). Also, I think it would be good to keep with the rest of the tutorial document's format and continue using the interpreter prompts. Lastly, if a new user copy pastes the code directly, and an actual OSError occurs, the example would silence it without the user knowing. I'd suggest a print to notify the user.
```suggestion
   >>> try:
   ...     f = open('workfile', 'w')
   ... except OSError as e:
   ...     print('File open failed')
   ... 
   
```

Reply:
Good points, thank you.

Reply:
should we also use the exception in the print message?

Reply:
I wondered about that too, but I'm mildly afraid that something like `print(f'File open failed, error {e}')`, would lead to a beginner not understanding f-strings to become confused. Maybe just a simple `print(e)`? Your call for this one, I'm not too sure :).

Reply:
I think print('Cannot open file: ', e) will do.